crates/bevy_app/src/app.rs

```diff
@@ -11,7 +11,7 @@ use bevy_ecs::{
     system::Resource,
     world::World,
 };
```

Comment:
This PR looks buggy. 

Looks like it breaks my project: https://github.com/LarsDu/StarRust/tree/bevy-0.9-unstable